ynr/apps/sopn_parsing/tests/test_sopn_helpers.py

```diff
@@ -23,51 +19,52 @@ def test_sopn_document(self):
         example_doc_path = abspath(
             join(dirname(__file__), "data/sopn-berkeley-vale.pdf")
         )
-        doc = SOPNDocument(example_doc_path)
+        doc = SOPNDocument(open(example_doc_path, "rb"))
         self.assertSetEqual(
             doc.document_heading,
             {
-                "",
-                "william",
                 "following",
-                "february",
                 "council",
-                "lindsey",
-                "mike",
                 "of",
                 "is",
-                "name",
                 "statement",
-                "candidate",
                 "district",
                 "the",
-                "little",
-                "vale",
-                "moscow,",
-                "2019",
-                "jane",
-                "as",
-                "willetts",
-                "stayte",
-                "on",
                 "for",
-                "berkeley",
                 "stroud",
-                "ashton",
-                "green",
-                "28",
-                "thursday",
-                "edward",
-                "berrycr",
                 "persons",
-                "liz",
                 "nominated",
                 "a",
-                "thomas",
-                "home",
-                "address",
                 "election",
                 "councillor",
+                "vale",
+                "berkeley",
+                "on",
+                "as",
+                "name",
+                "thursday",
+                "candidate",
+                "february",
+                "28",
+                "ashton",
+                "2019",
+                "labour",
+                "any)",
+                "simpson",
+                "jane",
+                "eleanor",
+                "why",
+                "description",
+                "(if",
+                "reason",
+                "green",
+                "party",
+                "liz",
+                "proposer",
+                "lindsey",
+                "longer",
+                "no",
+                "simpson",
```

Comment:
Just reading this test in isolation, its not exactly clear why this particular word soup is desirable. Just for future sanity, it might be useful to:

a) add a comment explaining what we do/don't expect to extract from the document.
b) group the words like they appear on the PDF. For example, if this was laid out like:

```py
self.assertSetEqual(
    doc.document_heading,
    {
        "berkeley",
        "vale",
        "on",
        "thursday",
        "28",
        "february",
        "2019",

        # table
        "name",
        "of",
        "candidate",
        #...etc

        # names
        "liz",
        "ashton",
        "lindsey",
        "jane",
        "green",
        #...etc
    }
)
```

it would make it way easier to understand what is supposed to be happening here (I appreciate this is a bit of a faff).